scout/server/blueprints/panels/templates/panels/panel.html

```diff
@@ -259,7 +259,7 @@ <h5 class="modal-title" id="causativeLabel">Export STR and SMN panel hits for a
         </div>
         <div class="modal-footer">
           <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-          <button name="action" value="export" type="submit" class="btn btn-primary" title="Mark causative">
+          <button name="action" value="export" type="submit" class="btn btn-primary" title="Export STR and SMN panel hits">
```

Comment:
Oh that's weird! Perhaps remove entirely the title thing, It doesn't look that is used

Reply:
Perhaps "export" was clear enough, or perhaps not many know about the tooltips. They can be a bit tricky to trigger on the modals - one has to hover just right..
![Screenshot 2024-02-19 at 10 17 52](https://github.com/Clinical-Genomics/scout/assets/758570/808619f6-07cd-4044-8670-d7f85ad94770)
Since it can be shown, and they are present on many buttons, I think I'll keep it.